tests/docker/wp-dev-lib/Dockerfile

```diff
@@ -0,0 +1,36 @@
+ARG WP_PHP_VERSION=7.4
```

Comment:
We're keeping it here for now and then moving to wp-dev-lib repo and eventually the Docker Hub.